test/InterchainTokenService.js

```diff
@@ -1179,14 +1179,39 @@ describe('Interchain Token Service', () => {
     describe('Send Token', () => {
         const amount = 1234;
         const destAddress = '0x5678';
-        let token, tokenId;
+        let token, tokenId, tokenManager;
+        let payload, wrappedPayload, payloadHash, transferToAddress;
+
+        const buildPaylods = async () => {
```

Comment:
```suggestion
        const buildPayload = async () => {
```
instead of this, create a separate encodeITSTransfer util that takes and returns the appropriate args